otftotfm/automatic.cc

```diff
@@ -746,7 +763,16 @@ update_autofont_map(const String &fontname, String mapline, ErrorHandler *errh)
             if (slash >= 0)
                 filename = filename.substring(slash + 1);
             String redirect = verbose ? " 1>&2" : " >" DEV_NULL " 2>&1";
-            String command = updmap_prog + " --nomkmap --enable Map " + shell_quote(filename) + redirect
+#if defined(W32TEX)
+// jtex_filetype is defined only in W32TeX
+            char *p = kpsei_var_value("jtex_filetype");
+            if (p != NULL) { // W32TeX
+                free(p);
+                String option = "--add ";
+            } else // TeXLive
+#endif
+            String option = "--enable Map ";
+            String command = "updmap --nomkmap " + option  + shell_quote(filename) + redirect
```

Comment:
This is not right. It won't even compile if W32TEX is defined.

Reply:
Something like that would compile, because in the current code when `W32TEX` is defined, `String option` is never in scope in the definition of `String command` where it is used.

Reply:
The thing is that obvious compilation errors under `W32TEX` make me wonder whether anyone needs `W32TEX` support for this program.